### ISSUE TITLE
Remove app insights and add in rotating log files

### ISSUE THREAD

**Issue by mvdicarlo** [mvdicarlo/postybirb#319]
I have decided against using Application Insights.
It would be a nice to have, but ultimately doesn't feel right for the app.

Should remove the app insights and do standard rotating log file.

**Comment by dalacquar** [comment]
Hello @mvdicarlo my name is Rafael and I am a student at the Federal Technological University of Paraná in Brazil. I am currently studying a subject in which I need to make some contributions to open source repositories. I wonder if you could give me some tasks so I can help?

**Comment by Kelketek** [comment]
@dalacquar The best way to move forward here would be to go to an issue, comment that you're planning to work on that issue, and then start doing so. If you have an idea how you'll approach it, it would be good to mention it there.

If there's a reason you *shouldn't* work on the task, someone should pipe up.

Once your contribution is ready, create a pull request and someone will review it and give you feedback, or else merge it.

**Comment by Kelketek** [comment]
Note-- this issue probably isn't the best one to do since we're having conversations about it right now internally. If you want to actively get involved with developer discussion, join the discord: https://discord.gg/jK5JQJF
